common.gradle

```diff
@@ -2,7 +2,7 @@ android {
     compileSdk 34
 
     defaultConfig {
-        minSdk 21
+        minSdk 26
```

Comment:
This removes a major portion of our user base. Please do not bump the version and instead find another way